test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -176,15 +191,22 @@ class test_ucp_sockaddr : public ucp_test {
         ucs_time_t time_limit = ucs_get_time() + ucs_time_from_sec(UCP_TEST_TIMEOUT_IN_SEC);
 
         while ((receiver().get_num_eps() == 0) && (ucs_get_time() < time_limit)) {
-            check_events(sender().worker(), receiver().worker(), wakeup);
+            check_events(sender().worker(), receiver().worker(), wakeup, NULL);
         }
     }
 
     void client_ep_connect(struct sockaddr *connect_addr)
     {
         ucp_ep_params_t ep_params = ucp_test::get_ep_params();
         ep_params.field_mask      |= UCP_EP_PARAM_FIELD_FLAGS |
-                                     UCP_EP_PARAM_FIELD_SOCK_ADDR;
+                                     UCP_EP_PARAM_FIELD_SOCK_ADDR |
+                                     UCP_EP_PARAM_FIELD_ERR_HANDLING_MODE |
```

Comment:
since need to handle the long worker address, yes.

Reply:
Pls add clear `TODO` comment why it's needed and should it be deleted or not. Why does the long worker address require `UCP_ERR_HANDLING_MODE_PEER`? Can it be handled as a local error? `MODE_PEER` may affect performance, we need to avoid such limitations when possible.